File: app/src/main/java/com/estrrado/tradopay/ui/modules/explore/ExploreOpenActivity.java
package com.estrrado.tradopay.ui.modules.explore;

import android.os.Bundle;

import com.estrrado.tradopay.ui.modules.MainActivity;
import com.estrrado.tradopay.ui.utilities.ActivityUtils;
import com.google.android.material.floatingactionbutton.FloatingActionButton;
import com.google.android.material.snackbar.Snackbar;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import android.view.MenuItem;
import android.view.View;
import android.widget.Toast;

import com.example.tradopay.R;

public class ExploreOpenActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_explore_open);
        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);
        ActivityUtils.set_back_button(ExploreOpenActivity.this,true);


    }

    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
        if(item.getItemId()==android.R.id.home)
        {
            onBackPressed();
        }
        return super.onOptionsItemSelected(item);
    }

    @Override
    public void onBackPressed() {
        super.onBackPressed();
        //Toast.makeText(ExploreOpenActivity.this,"getIntent_getComponent="+getIntent().,Toast.LENGTH_LONG).show();
    }
}

File: app/src/main/java/com/estrrado/tradopay/activity/ExploreOpenActivity.kt
package com.estrrado.tradopay.ui.modules.explore

import android.os.Bundle
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import com.estrrado.tradopay.activity.BaseActivity
import com.estrrado.tradopay.helpers.ActivityUtils

import com.example.tradopay.R

class ExploreOpenActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

    }

    override fun initLayout() {
        setContentView(R.layout.activity_explore_open)
        val toolbar = findViewById<Toolbar>(R.id.toolbar)
        setSupportActionBar(toolbar)
        ActivityUtils.set_back_button(this@ExploreOpenActivity, true)
    }

    override fun initListeners() {

        TODO("Not yet implemented")
    }

    override fun initObservers() {
        TODO("Not yet implemented")
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        if (item.itemId == android.R.id.home) {
            onBackPressed()
        }
        return super.onOptionsItemSelected(item)
    }

    override fun onBackPressed() {
        super.onBackPressed()
        //Toast.makeText(ExploreOpenActivity.this,"getIntent_getComponent="+getIntent().,Toast.LENGTH_LONG).show();
    }
}